modules/configuration/src/test/java/org/apache/ignite/internal/configuration/notifications/ConfigurationAnyListenerTest.java

```diff
@@ -104,7 +104,7 @@
     private RootConfiguration rootConfig;
     
     /** Notification events. */
-    private final List<String> events = new ArrayList<>();
+    private final List<String> events = new CopyOnWriteArrayList<>();
```

Comment:
Better to use `ConcurrentLinkedQueue`

Reply:
Why? The original code uses a `List`, I did not want to change it because the performance is not actually important here. I just wanted to use something thread-safe.

Reply:
If you look at it, then there does not have to be a `List`.

Reply:
This would require me to change more code which I didn't touch at all. Are you sure we need to make this improvement now? As I don't even see why it's needed :)

Reply:
Then I suggest not to change the code, since it hasn't broken yet.

Reply:
Well, here a non-thread-safe collection is used in the context where writes could come from different threads. This looks like a bug. Why wait till someone hits it if something changes inside configuration changer mechanism?